superbench/common/utils/process.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
The code logic looks like the same, correct?

Reply:
Yes, code logic remain unchanged.